### PR TITLE
readme: clarify what's not in homebrew/brew

### PR DESCRIPTION
**Pull request by dunn**
cc @nijikon @mikemcquaid

### CODE REVIEW

**Comment by tdsmith** [comment]
careful; you pushed a branch to Homebrew/legacy-homebrew :p

File: README.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Homebrew (Legacy)
+
```
Comment (mikemcquaid) [inline README.md]: NOPE, don't need this 😛 (sorry, I hate unnecessary Markdown whitespace)

File: README.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
...
 This repository has been deprecated and split into two repositories:
-- [Homebrew/brew](https://github.com/Homebrew/brew): the Homebrew package manager without any formulae/packages (i.e. the former contents of `Library/Formula`)
+- [Homebrew/brew](https://github.com/Homebrew/brew): the Homebrew
+  package manager without any formulae/packages (i.e., without the former contents of `Library/Formula`)
```
Comment (mikemcquaid) [inline README.md]: `i.e.,` doesn't need a comma.

File: README.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
...
+
```
Comment (mikemcquaid) [inline README.md]: Don't need the newline.

**Comment by mikemcquaid** [comment]
a few comments but otherwise 👍

**Comment by mikemcquaid** [comment]
Thanks @dunn!

**Pull request closed without merging**